src/commands/spec.js

```diff
@@ -9,6 +9,7 @@ const cmd = {
     run: async toolbox => {
 
         const herbarium = requireHerbarium("spec", toolbox.filesystem.cwd())
+        herbarium.requireAll()
```

Comment:
Why is it necessary? It was working without it, right?

Reply:
Using Herbarium it was necessary to perform requireAll, without it it is not possible to read the test files and useCases.